src/tcgwars/logic/impl/gen4/SecretWonders.groovy

```diff
@@ -2769,14 +2784,27 @@ public enum SecretWonders implements LogicCardInfo {
           pokePower "Teleportation", {
             text "Once during your turn , choose 1 of your Active Pokémon or 1 or your Benched Pokémon and switch Gardevoir with that Pokémon. This power can’t be used if Gardevoir is affected by a Special Condition."
             actionA {
```

Comment:
You need to check that Gardevoir doesn't have a status condition affecting it when using this power.

Reply:
Fixed

Reply:
Looks like `checkNoSPC()` is still missing from this ability